go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -151,8 +151,36 @@ func (se *Engine) syncVTDatabase(ctx context.Context, conn *dbconnpool.DBConnect
 		}
 		return conn.ExecuteFetch(query, maxRows, wantFields)
 	}
+	var _ sidecardb.ReSetSuperReadOnlyHook = func(ctx context.Context) (err error) {
+		log.Infof("resetting hook for super read only...")
+		if _, err := dbaConn.ExecuteFetch("SET GLOBAL super_read_only='ON'", 1, false); err != nil {
+			log.Infof("Not able to set super_read_only user... This can cause errant GTIDs in future.")
+			return err
+		}
+		return nil
+	}
+	var _ sidecardb.SetSuperReadOnlyHook = func(ctx context.Context) (needsReset bool, err error) {
+		if !dbaConn.IsMariaDB() {
```

Comment:
Same comment here as elsewhere about not performing the requested action but not indicating a failure or any problem.